### ISSUE TITLE
`Popup`  window aspect artifacts

### ISSUE THREAD

**Issue by acgc99** [acgc99/Godot-AppComponents#19]
This is an scaling artifact caused by `display/window/stretch/aspect = expand`, with `ignore`/`keep` there is no problem. Related to #18. This can be solved with anchors.

![image](https://github.com/acgc99/Godot-AppComponents/assets/96864661/ae88335c-d767-4dd5-bd06-4ea44451db73)

![image](https://github.com/acgc99/Godot-AppComponents/assets/96864661/914d7cea-e3c9-445d-beb0-2af7dfa2a0a9)

**Comment by acgc99** [comment]
This can be solved with anchors I hope, like #18. I decided to use appearing/disappearing animations to bypass the problem, I am not very fluent with anchors and I could not be able to get a full solution.

**Comment by acgc99** [comment]
Now I recovered shrink/expand behavior. You have to anchors preset to full rest and prepare position and scale manually at the scene where the popup is placed:
```
func _ready() -> void:
	popup.position = size/2.0
	popup.scale = Vector2.ZERO
```
